lib/getStatType.m

```diff
@@ -0,0 +1,37 @@
+%==========================================================================
+%Obtain what type of statistic the NIDM-Pack uses (i.e. 'T', 'F', 'Z', 'P'
+%or 'X' as well as the statisticMaps objects stored inside the NIDM-Results
+%pack. The function takes in one input:
+%
+%graph - the graph from inside an NIDM-Results json.
+%
+%Authors: Thomas Maullin, Camille Maumet.
+%==========================================================================
+
+function [type, statisticMaps] = getStatType(graph)
```

Comment:
Accounting for different statistics maps and contrast maps is going to be a large task - can I suggest we address all of the multiple map changes in a separate PR?